nuxeo-core/nuxeo-core-bulk/src/test/java/org/nuxeo/ecm/core/bulk/action/TestWordCountAction.java

```diff
@@ -157,6 +160,41 @@ public void testWordCountWithLimitedQuery() throws Exception {
         assertEquals(wordCount, result);
     }
 
+    @Test
+    public void testTransactionalSubmit() throws Exception {
+        int wordCount = 42;
+        String myFile = createFile(wordCount);
+        BulkCommand command = new BulkCommand.Builder("testWordCountLimited", myFile).useGenericScroller().build();
+        String commandId = bulkService.submitTransactional(command);
+        // status of bulk command is unknown until committed
+        BulkStatus status = bulkService.getStatus(commandId);
+        assertEquals(UNKNOWN, status.getState());
+        // commit
+        TransactionHelper.commitOrRollbackTransaction();
+        TransactionHelper.startTransaction();
+        status = bulkService.getStatus(commandId);
+        assertNotEquals(UNKNOWN, status.getState());
+        assertTrue(bulkService.await(commandId, Duration.ofSeconds(20)));
+        status = bulkService.getStatus(commandId);
+        assertEquals(COMPLETED, status.getState());
+        assertEquals(wordCount, status.getResult().get("wordCount"));
+
+        // Rollback case
+        BulkCommand commandBis = new BulkCommand.Builder("testWordCountLimited", myFile).useGenericScroller().build();
+        commandId = bulkService.submitTransactional(commandBis);
+        // state of bulk command is unknown until committed
+        status = bulkService.getStatus(commandId);
+        assertEquals(UNKNOWN, status.getState());
+        // rollback
+        TransactionHelper.setTransactionRollbackOnly();
+        TransactionHelper.commitOrRollbackTransaction();
+        TransactionHelper.startTransaction();
+        // No bulk command executed
+        status = bulkService.getStatus(commandId);
+        assertEquals(UNKNOWN, status.getState());
+    }
+
+
```

Comment:
extra new line 🥒

Reply:
yep, but I will ignore it to save a 4h build.

Reply:
thug life